### PR TITLE
kim-api 2.0.2 (new formula) & openkim-models 2019-03-29 (new formula)

### PR DESCRIPTION
**Pull request by ellio167**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by ellio167** [comment]
Any thoughts on accepting this?

Once accepted, the kim-api package will be added as a dependency to the next release of the `lammps` package.

**Comment by ellio167** [comment]
@fxcoudert is this something you would be willing to look at?

**Comment by ellio167** [comment]
These packages are now available as official updates in the Fedora 28, 29, and 30 releases:

https://apps.fedoraproject.org/packages/kim-api
https://apps.fedoraproject.org/packages/openkim-models

It would be great to have them available for macOS too.

**Comment by ellio167** [comment]
@smillerdev does the ‘maintainer feedback’ label mean I need to do something? Or is this a request for the homebrew maintainers to respond to this PR?

**Comment by SMillerDev** [comment]
> @SMillerDev does the ‘maintainer feedback’ label mean I need to do something? Or is this a request for the homebrew maintainers to respond to this PR?

The latter, unfortunately it's just some more waiting for you.

**Comment by ellio167** [comment]
@smillerdev thanks for the reply and clarification!

File: Formula/kim-api.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    args << "-DKIM_API_SYSTEM_MODELS_DIR=:#{HOMEBREW_PREFIX}/lib/openkim-models/models"
+    args << "-DKIM_API_SYSTEM_SIMULATOR_MODELS_DIR=:#{HOMEBREW_PREFIX}/lib/openkim-models/simulator-models"
+    system "cmake", ".", *args
+    system "make"
+    system "make", "docs"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/kim-api-collections-management list > ./out"
```
Comment (fxcoudert) [inline Formula/kim-api.rb]:
```suggestion
    output = shell_output("#{bin}/kim-api-collections-management list")
    assert_match "ex_model_Ar_P_Morse_07C_w_Extensions", output
```

File: Formula/openkim-models.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    args = std_cmake_args
+    args << "-DKIM_API_MODEL_DRIVER_INSTALL_PREFIX=#{lib}/openkim-models/model-drivers"
+    args << "-DKIM_API_MODEL_INSTALL_PREFIX=#{lib}/openkim-models/models"
+    args << "-DKIM_API_SIMULATOR_MODEL_INSTALL_PREFIX=#{lib}/openkim-models/simulator-models"
+    system "cmake", ".", *args
+    system "make", "install"
+  end
+
+  test do
+    system "kim-api-collections-management list > ./out"
```
Comment (fxcoudert) [inline Formula/openkim-models.rb]:
```suggestion
    output = shell_output("kim-api-collections-management list")
    assert_match "LJ_ElliottAkerson_2015_Universal__MO_959249795837_003", output
```

**Review by fxcoudert — CHANGES_REQUESTED** [review]
Technically, this is good. The question is really one of notability: this seems very little-used (see stars/forks metrics on https://github.com/openkim/kim-api). I'm not sure this is suitable for Homebrew-core, but would be more appropriate for an organisation tap, at least in a first time: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

**Comment by ellio167** [comment]
@fxcoudert Thanks very much for your response!  I understand your concern.  Please let me make one argument for the inclusion of these packages.

LAMMPS is already included in homebrew-core and the kim-api will soon become a standard dependency of LAMMPS.  With the latest release (05 Jun 2019) binary distributions of lammps are now being linked against libkim-api.  See for just one example, https://fedora.pkgs.org/30/fedora-updates-x86_64/lammps-20190605-3.fc30.i686.rpm.html.

For this reason, I would argue that it makes sense to include these packages in homebrew-core.  If accepted, I would immediately submit a PR to the lammps formula with an update to the latest (05 Jun 2019) stable release and settings to build with the kim-api library.

If this argument is not sufficiently convincing, then we'll accept the maintainers' decision and create an organization tap.  However, this will lead us to duplicate the lammps formula within our tap and seems inefficient to me.

Thanks for your attention and for providing such a great tool.  It is very much appreciated.  I hope we can contribute to its ongoing success.

**Comment by fxcoudert** [comment]
That sounds like a good reason, so let's include this.
Can you please squash things so we have a single commit per formula? Thanks.

**Comment by ellio167** [comment]
Great!!!  I've squashed.

**Pull request closed without merging**